Revamp tfe sidebar (#2324)
hashicorp/terraform-website: content/enterprise/requirements/hardware.mdx

---
page_title: Hardware and Data Storage - Requirements - Terraform Enterprise
description: >-
  Required disk space and memory for your Terraform Enterprise instance.
---

# Hardware and Data Storage Requirements

> It is critical that you set up the correct hardware before installing and running Terraform Enterprise. Incorrect amounts of disk space or memory can cause significant performance issues.

Reviewer: ```suggestion
You must set up the correct hardware before installing and running Terraform Enterprise. Incorrect amounts of disk space or memory can cause significant performance issues.
```
